### PR TITLE
Add cache hits summary to ftpscraper

### PR DESCRIPTION
**Pull request by willkg**
This helps us know whether the file-based cache is helping or not without being
overly verbose.

### CODE REVIEW

**Comment by willkg** [comment]
I wanted some kind of indicator that the cache is getting hit and for how much. Originally, I had it printing "hit!" every time, but agreed that's a bit much. This changes it to a final count at the end.

@peterbe Is this ok with you? If not, I can figure out another way to do it.

**Comment by willkg** [comment]
I like that idea. ^^^ improves it by adding cache misses and percentages. It looks like this:

```
2017-09-07 20:12:50,209 DEBUG - MainThread -  - MainThread - Cache: hits: 608 (1.00%) misses: 0 (0.00%)
```

(That's after running it twice in a row.)

File: socorro/cron/jobs/ftpscraper.py [hunk 1]
```diff
@@ -368,7 +370,13 @@ def run(self, date):
...
 
         if self.config.cachedir:
-            self.config.logger.debug('Cache hits: %d' % self.cache_hits)
+            total = float(self.cache_hits + self.cache_misses)
+            self.config.logger.debug('Cache: hits: %d (%2.2f%%) misses: %d (%2.2f%%)' % (
+                self.cache_hits,
+                self.cache_hits / total * 100,
+                self.cache_misses,
+                self.cache_misses / total * 100,
+            ))
```
Comment (peterbe) [inline socorro/cron/jobs/ftpscraper.py]: It's not like you didn't know this but one convenient way to do percentages in python 2.7 of ints is to do:
```
p = 100.0 * one_int / another_int
```

**Comment by peterbe** [comment]
Still approve.

**Comment by willkg** [comment]
Thank you!

**Pull request merged**